docs/source/extensions/strategy.rst

```diff
@@ -0,0 +1,111 @@
+.. _strategy:
+
+########
+Strategy
+########
+
+Strategy depicts the training strategy to be used by the :doc:`Lightning Trainer <../common/trainer>`. It can be controlled by passing different
+training strategies with aliases (``ddp``, ``ddp_spawn``, ``deepspeed``, etc) as well as custom training strategies to the ``strategy`` parameter for Trainer.
+
+Strategy in Lightning handles some of the following responsibilities:
```

Comment:
I think a gif would make the explanation much simpler.

Reply:
I agree! @rohitgr7 I think we could audit the docs again where visual content could benefit.